guests/templates/checklist/merch_deadline.html

```diff
@@ -31,7 +31,7 @@
   <form method="post" action="merch" class="form-horizontal" role="form" enctype="multipart/form-data">
     {{ csrf_token() }}
     <input type="hidden" name="guest_id" value="{{ guest.id }}" />
-    <input type="hidden" name="id" value="{{ guest_merch.db_id }}" />
+    <input type="hidden" name="id" value="{{ guest_merch.db_id or 'None' }}" />
```

Comment:
If `guest_merch` is `None` then `db_id` is never called. We need this value to actually be the string `"None"`.